src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -118,13 +155,29 @@ const ChartsHolder = (props: {
               onContactTracingClick={() => scrollTo(contactTracingRef.current)}
             />
             <MainContentInner>
-              <ChartHeader ref={rtRangeRef}>
-                {getMetricName(Metric.CASE_GROWTH_RATE)}
-              </ChartHeader>
+              <ChartHeaderWrapper>
+                <ChartHeader ref={rtRangeRef}>
+                  {getMetricName(Metric.CASE_GROWTH_RATE)}
+                </ChartHeader>
+                {!isMobile && rtRangeData && (
+                  <ShareButtons
```

Comment:
I'm not wild about all the repetition going on (10 of these ShareButtons instantiations).  I think we could clean it up a bit by creating a `metricToShareButtons` map up top or something, but we should probably rework this whole page to be more componentized rather than repeat ourselves for each metric.  So we can punt for now.